velox/experimental/cudf/exec/ExpressionEvaluator.cpp

```diff
@@ -837,6 +1017,15 @@ bool registerCudfFunction(
   return true;
 }
 
+void registerCudfFunctions(
```

Comment:
This is great. Can you also replace other multiple function registrations in this file with this aliases based one?